pyinstaller/README.md

```diff
@@ -2,24 +2,70 @@
 
 Install `pyinstaller`:
 ```bash
-$ pip3 install pyinstaller
+$ pip3 install -r requirements.txt
 ```
 
 `cd` into this directory (`specter-desktop/pyinstaller`) and run:
```

Comment:
This should be on top of the file before the `pip3 install`

Reply:
done :)